CalcualtorLogic.js

```diff
@@ -28,6 +28,9 @@ class Calculator {
     constructor(selector) {
         this.hostElem = document.querySelector(selector);
     };
-    render() {};
+    render() {
+        const wrapperDiv = document.createElement('div');
+        wrapperDiv.className = 'wrapper';
```

Comment:
please Use `classlist` api to add the class

Example:
![image](https://user-images.githubusercontent.com/16171515/108386326-726b4380-7204-11eb-9898-67cc9231bd9b.png)